src/analysis.cpp

```diff
@@ -127,21 +127,6 @@ namespace Faunus
         f << energy() << "\n"; 
     }
 
-    void PairFunctionBase::normalize(data &d)
```

Comment:
Why has this been removed from the base class and implemented separately in each derived class, i.e. `AtomRDF` and `MoleculeRDF`?